prow/config.yaml

```diff
@@ -2065,9 +2065,9 @@ presubmits:
         - "--clean"
         - "--git-cache=/root/.cache/git"
         - "--job=$(JOB_NAME)"
-        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--repo=github.com/kubernetes-security/$(REPO_NAME)=$(PULL_REFS)"
```

Comment:
In some places `${REPO_NAME}` seems to be 'org/repo' format and in others 'org' (like here). Is the format variable?

Reply:
REPO_NAME should be `repo` for presubmits AFAIK, and for all of these cases we had `k8s.io/repo` which should be `github.com/kubernetes-security` for the security jobs.

Reply:
It seems to be `k8s.io/repo` format elsewhere: https://github.com/kubernetes/test-infra/pull/5804/files#diff-406185368ba7839d1459d3d51424f104R2018

Are those places incorrect too?

Reply:
no, there are other repos like k8s.io/release that aren't kubernetes/kubernetes

Reply:
the presubmit should cover the necessary changes now, if it doesn't somehow we'll patch it in a follow-up, you can see that the tests fail once the new presubmit is introduced and then pass once the config changes are added so that part should be correct at least.

As far as which flags needed updating:
- flags that are effectively `--repo=github.com/kubernetes/kubernetes.*` need to to effectively be `--repo=github.com/kubernetes-security/kubernetes.*`
- flags that are `--upload=gs://some-bucket` need to be the private security bucket instead